Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.yml

```diff
@@ -1262,10 +1262,14 @@ script:
   - name: vt-passive-dns-data
     description: Returns passive DNS records by indicator.
     arguments:
-    - name: ip
-      required: true
+    - name: id
+      required: false
       default: true
+      description: IP or domain for which to get its DNS data.
+    - name: ip
```

Comment:
This might break backwards in case a user is using this command and passes an IP as a default argument